go.mod

```diff
@@ -44,6 +44,11 @@ require (
 	google.golang.org/protobuf v1.27.1
 )
 
+require (
+	github.com/jessevdk/go-flags v1.4.0
+	gopkg.in/yaml.v2 v2.4.0
+)
```

Comment:
Can you merge this with the block above?

Reply:
this is what `go mod tidy` does automatically and I'm not sure why.  I'll try it and see if it complains.